dom.js

```diff
@@ -6,3 +88,113 @@ function displayMenu() {
+const openButton5 = document.querySelector('#btn4');
+const openButton6 = document.querySelector('#btn5');
+
+function displayPopup() {
+  const blur = document.querySelector('.header');
+  blur.classList.toggle('active');
+  const blur1 = document.querySelector('.principal');
+  blur1.classList.toggle('active');
+  const blur2 = document.querySelector('.work');
+  blur2.classList.toggle('active');
+  const blur3 = document.querySelector('.cards');
+  blur3.classList.toggle('active');
+  const blur4 = document.querySelector('.about');
+  blur4.classList.toggle('active');
+  const blur5 = document.querySelector('.skill');
+  blur5.classList.toggle('active');
+  const blur6 = document.querySelector('.form');
+  blur6.classList.toggle('active');
+  const blur7 = document.querySelector('footer');
+  blur7.classList.toggle('active');
+  const div = document.createElement('div');
+  div.className = 'popup';
+
+  for (let i = 0; i < workcards.length; i += 1) {
+    if (workcards[i].id === id) {
+      div.innerHTML = `
+      <h2 class="pop-title">${workcards[i].titulo}</h2>
+      <div>
+        <ul class="techno">
+          <li class="techno-pop">${workcards[i].lenguaje}</li>
+          <li class="techno-pop">${workcards[i].lenguaje2}</li>
+          <li class="techno-pop">${workcards[i].lenguaje3}</li>
+        </ul>
+      </div>
+      <div class="desktop-pop-row">
+        <div class="pop-img"></div>
+        <div class="desktop-pop-col">
+          <div class="pop-text">
+            <p>
+              ${workcards[i].textpop}
+            </p>
+          </div>
+          <div>
+            <ul class="pop-buttons">
+              <li class="see one"><a href="${workcards[i].deploy}" target="_blank">See Live</a></li>
+              <li class="see two"><a href="${workcards[i].profile}" target="_blank">See Sourse</a></li>
+            </ul>
+          </div>
+        </div>  
+      </div>
+    `;
+    }
+  }
+  body.appendChild(div);
+  const cross = document.querySelector('.pop-title');
+  cross.addEventListener('click', () => {
+    body.removeChild(div);
+    blur.classList.toggle('active');
+    blur1.classList.toggle('active');
+    blur2.classList.toggle('active');
+    blur3.classList.toggle('active');
+    blur4.classList.toggle('active');
+    blur5.classList.toggle('active');
+    blur6.classList.toggle('active');
+    blur7.classList.toggle('active');
+  });
+}
+
+openButton.addEventListener('click', () => {
+  id = 0;
+  displayPopup();
+});
+
+openButton1.addEventListener('click', () => {
+  id = 1;
+  displayPopup();
+});
+
+openButton2.addEventListener('click', () => {
+  id = 2;
+  displayPopup();
+});
+
+openButton3.addEventListener('click', () => {
+  id = 3;
+  displayPopup();
+});
+
+openButton4.addEventListener('click', () => {
+  id = 4;
+  displayPopup();
+});
+
+openButton5.addEventListener('click', () => {
+  id = 5;
+  displayPopup();
+});
+
+openButton6.addEventListener('click', () => {
+  id = 6;
+  displayPopup();
+});
```

Comment:
- [ ] [OPTIONAL] When I click on the `See project` button, the Popup section is displayed on the Homepage. Please show the Popup in the middle of the screen when you click the `See Project` button.
#
![image](https://user-images.githubusercontent.com/63915024/181389331-ed547238-a686-43ef-8d5c-32f420b22f0d.png)
#

Reply:
Ok @mukhammadrizooff !!, thanks i will do that 👍